test/e2e/tests/test_layer_version.py

```diff
@@ -103,6 +103,11 @@ def test_smoke(self, lambda_client):
         _, deleted = k8s.delete_custom_resource(ref)
         assert deleted is True
 
+        # Check if all versions are deleted
+        layer_name = cr['spec']['layerName']
+        list = lambda_validator.list_layer_versions(layer_name)
+        assert len(list["LayerVersions"]) == 0
```

Comment:
Are multiple layers created during the tests?

Reply:
There are multiple versions created for a single LayerVersion. The `list` here contains all those versions of a LayerVersion `layer_name`.

Reply:
Sure, but does the test create many versions? Are we actually testing the new code with the existing resource setup?

Reply:
@RedbackThomson Yes, updating a LayerVersion (even the description) will bump the layer version..